evasquez26/angular4-startup
# S6-ONE-226 Bug: Error on existing email on registration

## Story / Task


see [ONE-226](https://bixlabs.atlassian.net/browse/ONE-226)


## What this PR does?

Fixes bug: When trying to register with an existing email, a generic message is shown

### Before 

When trying to register with an existing email, a generic message is shown

### After

An specific message is shown
![captura de pantalla de 2017-06-01 23 11 40](https://cloud.githubusercontent.com/assets/2192121/26711720/b30e7754-471f-11e7-8afc-bc4f77237815.png)




### How to test / reproduce

- Logout from the app
- Try to register with an existing email
- Check the error message

### TODOs

### Related PRs
N/A

## Migrations

NO


Thank you.
